pkgs/development/python-modules/bip_utils/default.nix

```diff
@@ -4,22 +4,39 @@
 , pythonOlder
 , ecdsa
 , pysha3
+, coincurve
+, pynacl
+, crcmod
+, ed25519-blake2b
+, py-sr25519-bindings
+, cbor2
+, pycryptodome
 }:
 
 buildPythonPackage rec {
   pname = "bip_utils";
-  version = "2.5.1";
+  version = "2.7.0";
 
   disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "ebellocchia";
     repo = pname;
-    rev = "refs/tags/v${version}";
-    sha256 = "sha256-lH8hd+JA1FhGH60MYIIuwHjr/4wFbYeuw/hd60kr1xc=";
+    rev = "v${version}";
+    sha256 = "sha256-m7/CC5/T6qR2Ot4y5WQlzOAR0czz6XHCjJskES+2nns=";
   };
 
-  propagatedBuildInputs = [ ecdsa pysha3 ];
+  propagatedBuildInputs = [
+    ecdsa
+    pysha3
+    cbor2
+    pynacl
+    coincurve
+    crcmod
+    ed25519-blake2b
+    py-sr25519-bindings
+    pycryptodome
```

Comment:
How were we getting away without half of these before? The package's requirements don't appear to have changed a lot since 2.5.1.